python/lsst/ts/watcher/model.py

```diff
@@ -230,6 +230,37 @@ def get_rules(self, name_regex):
         compiled_re = re.compile(name_regex)
         return [rule for name, rule in self.rules.items() if compiled_re.match(name) is not None]
 
+    def mute_alarm(self, name, duration, severity, user):
+        """Mute one or more alarms for a specified duration.
+
+        Parameters
+        ----------
+        name : `str`
+            Alarm name.
+            If a regular expression then acknowledge all matching alarms.
```

Comment:
As in my other comment, please reword the "Alarm name" portion.

Reply:
Yes. I did this everywhere.